arangodb-net-standard/CursorApi/CursorApiClient.cs

```diff
@@ -97,7 +98,7 @@ public async Task<DeleteCursorResponse> DeleteCursorAsync(string cursorId)
 
         public async Task<PutCursorResponse<T>> PutCursorAsync<T>(string cursorId)
         {
-            using (var response = await _client.PutAsync(_cursorApiPath + "/" + cursorId, new StringContent(string.Empty)))
+            using (var response = await _client.PutAsync(_cursorApiPath + "/" + WebUtility.UrlEncode(cursorId), new StringContent(string.Empty)))
```

Comment:
This line is more than 100 characters. Should we shorten it as per our conventions?